src/metabase/query_processor/pivot/postprocess.clj

```diff
@@ -472,10 +489,9 @@
       into []
       (let [sections-rows
             (mapv (fn [section-row-combos]
-                    (into []
-                          (keep (fn [row-combo]
```

Comment:
This `keep` is being intentionally removed, right? I think you're correct here that it's not needed anymore since we shouldn't be generating extra rows in the first place, but just want to confirm.

Reply:
Yes, `build-row` always returns something now, so keep is no longer necessary.